app/screens/CfpScreen.tsx

```diff
@@ -133,6 +135,14 @@ const CfpScreen = ({ session }: { session?: Session }) => {
                           <CompactCell>ID</CompactCell>
                           <CompactCell>#{result.number}</CompactCell>
                         </CompactRow>
+                        <CompactRow>
+                          <CompactCell>{t("cfp.type")}</CompactCell>
+                          <CompactCell>{t(`${result.type.toUpperCase()}`)}</CompactCell>
+                        </CompactRow>
+                        {result.dfiAmount && <CompactRow>
+                          <CompactCell>{t("cfp.amount")}</CompactCell>
+                          <CompactCell>{t(`${result.dfiAmount}`)}</CompactCell>
```

Comment:
Maybe place the `DFI` in the right cell? So it looks like
`| Amount | 100 DFI |`